vaadinfrontend/src/main/java/life/qbic/datamanager/views/components/CardLayout.java

```diff
@@ -0,0 +1,243 @@
+package life.qbic.datamanager.views.components;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.dependency.CssImport;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.orderedlayout.FlexComponent;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+
+/**
+ * <b>Card Layout</b>
+ *
+ * <p>A card Component with a shadow containing customizable components such as a title, a layout
+ * for fields and a layout for buttons.
+ *
```

Comment:
Maybe a little bit more information about the different layouts:

As a user of this class, what do I need to know to format it? What can I expect the layout to look like? When I extend the class, that do I need to be aware of?